src/main/java/fruitShop/Main.java

```diff
@@ -0,0 +1,4 @@
+package fruitShop;
+
+public class Main {
+}
```

Comment:
This class is also shouldn't be here...

Reply:
Sure. It leaked to git despite all my attempts to prevent this). Thanks!